sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.csproj

```diff
@@ -0,0 +1,103 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+		<IsExtensionClientLibrary>true</IsExtensionClientLibrary>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Azure.WebJobs" />
+		<PackageReference Include="Microsoft.AspNetCore.Http" />
+		<PackageReference Include="Microsoft.IdentityModel.Protocols" />
+		<PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" />
+	</ItemGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
We should consider removing this and allowing the defaults from the repository.

Reply:
Done

Reply:
Done